test/trust.test.js

```diff
@@ -1,145 +1,148 @@
+import { DEFAULT_USER_LIMIT_PERCENTAGE } from '~/common/constants';
+
 import createCore from './helpers/core';
-import getAccount from './helpers/account';
 import getTrustConnection from './helpers/getTrustConnection';
-import web3 from './helpers/web3';
-import { deploySafeAndToken } from './helpers/transactions';
-
-let account;
-let otherAccount;
-let core;
-let safeAddress;
-let otherSafeAddress;
-
-beforeAll(async () => {
-  account = getAccount();
-  otherAccount = getAccount(3);
-  core = createCore();
-});
+import setupWeb3 from './helpers/setupWeb3';
+import getAccounts from './helpers/getAccounts';
+import setupAccount from './helpers/setupAccount';
+import generateSaltNonce from './helpers/generateSaltNonce';
 
 describe('Trust', () => {
-  beforeAll(() =>
-    Promise.all([
-      deploySafeAndToken(core, account),
```

Comment:
this is missing right? shall we add a TODO to dont forget

Reply:
Not really, this method was used because of the safe deployment, its not about the token when trusting. These tests are testing exactly the same thing.

Reply:
I mean it was there to test both deploy safe and token and then test trust, but somehow this has to live somewhere. The safedeployandtoken will you move it to safe?

Reply:
also trust is related to tokens, bc when you get trusted it means you can receive that person circles so in a way it's needed for the tests

Reply:
You're right, trusting means accepting tokens but in the tests of `trust` before the replacement there were no transfer whatsoever. They are just checking that a safe can trust and be trusted querying the subgraph. In the near future, we will need to add the token deployment as well for the `token.js` replacement. But i think that we can add that in the new helper method `setupAccount` that does that, deploys a safe with the new implementation and signs it in the hub.

Reply:
ok, that sounds good!  Although I would call it the same bc setupAccount is confusing in terms of what we mean by "account". Let's keep writing these todos somewhere, in our docs or in  list of todo so we dont forget